website/docs/r/cloud_project_kube.html.markdown

```diff
@@ -34,10 +46,19 @@ The following arguments are supported:
    Changing this value recreates the resource.
 
 * `version` - (Optional) kubernetes version to use.
-   Changing this value recreates the resource. Defaults to latest available.
+   Changing this value updates the resource. Defaults to latest available.
 
-* `private_network_id` - (Optional) OpenStack private network (or vrack) ID to use.
-   Changing this value recreates the resource. Defaults - not use private network.
+* `private_network_id` - (Optional) OpenStack private network ID to use.
+   Changing this value delete the resource(including ETCD user data). Defaults - not use private network.
+   
+   
+**WARNING: update private network id reset the cluster so all user data are deleted**
+
+* `private_network_configuration` - (Optional) The private network configuration
+  * default_vrack_gateway - If defined, all egress traffic will be routed towards this IP address, which should belong to the private network. Empty string means disabled.
+  * private_network_routing_as_default - Defines whether routing should default to using the nodes' private interface, instead of their public interface. Default is false.
+
+* `update_policy` - Cluster update policy. Choose between [ALWAYS_UPDATE,MINIMAL_DOWNTIME,NEVER_UPDATE]'.
```

Comment:
As it's possible to update this information (and not to add it when you create a cluster for the moment), I think we can keep this information in the documentation

```suggestion
* `update_policy` - Cluster update policy. Choose between [ALWAYS_UPDATE, MINIMAL_DOWNTIME, NEVER_UPDATE]'.
```